app/controllers/admin/finance/statements_controller.rb

```diff
@@ -46,9 +46,9 @@ def statements_query
         contract_period_years:,
         statement_date:,
         fee_type:,
-        sort: "+year,+month",
+        order_by: :statement_date,
```

Comment:
Is it worth passing the ordering as well here to be consistent with the route we're taking in the API? so `order_by: { statement_date: :asc }`